application/controllers/sql_queries.py

```diff
@@ -24,3 +25,7 @@ def query_instrument_by_id(instrument_id, session):
 
 def query_instrument(session):
     return session.query(InstrumentModel)
+
+
+def query_seft_instrument_by_instrument_id(instrument_id, session):
```

Comment:
is one() required? wont it only send one record back?

Reply:
I want it to throw an exception if it somehow got more than one back.  I don't think it's possible but theres no harm in the extra check